contrib/python-pybind11/template.py

```diff
@@ -0,0 +1,69 @@
+pkgname = "python-pybind11"
+pkgver = "2.11.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "cmake",
+    "pkgconf",
+]
+makedepends = [
+    "boost-devel",
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = ["python-pytest"]
+pkgdesc = "Seamless operability between C++11 and Python"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "BSD-3-Clause"
+url = "https://pybind11.readthedocs.io/en/stable/index.html"
+source = f"https://github.com/pybind/pybind11/archive/refs/tags/v{pkgver}.zip"
+sha256 = "b011a730c8845bfc265f0f81ee4e5e9e1d354df390836d2a25880e123d021f89"
+options = ["!cross"]
+
+
+def post_configure(self):
+    self.do("cmake", "-B", "build-tests")
```

Comment:
i'm pretty sure you don't need an entire second build directory to run tests? only `-DPYBIND11_TEST=ON`

Reply:
oh right, it's python by default with this being an extra

Reply:
you should use util.cmake for these then, like qbittorrent

though the installing is optional since it installs the same files just outside site-packages (which we symlinked below)

Reply:
Okay, yes I don't think cmake install is needed, the python install is what installs the python module as well.  So would just install things that are symlinked